--- fudgefont-1.4-src/SConstruct~	2008-12-23 15:16:24.000000000 +0100
+++ fudgefont-1.4-src/SConstruct	2008-12-23 15:17:35.000000000 +0100
@@ -68,8 +68,8 @@
     env.ParseConfig("freetype-config --libs")
     if static: env.Append(CCFLAGS = ["-DALLEGRO_STATICLINK"])
     CPPPATH.append("/usr/include/freetype2")
-    LIBDIR = "/usr/local/lib"
-    INCLUDEDIR = "/usr/local/include"
+    LIBDIR = "/usr/lib"
+    INCLUDEDIR = "/usr/include"
 
     library = env.SharedLibrary(libname,
         [BUILD + "/fudgefont.c",
